content-services/latest/install/ansible.md

```diff
@@ -24,16 +24,45 @@ There are two types of installations - local and remote:
 
 ## Prerequisites
 
-If you're using the Content Services (Enterprise), then you need credentials to access the necessary artifacts from [Nexus](https://artifacts.alfresco.com/nexus/){:target="_blank"}. Customers can request these through [Hyland Community](https://community.hyland.com/){:target="_blank"}.
+If you're using the Content Services (Enterprise), then you need credentials to access the necessary artifacts from 
+[Nexus](https://artifacts.alfresco.com/nexus/){:target="_blank"}. Customers can request these through 
+[Hyland Community](https://community.hyland.com/){:target="_blank"}.
 
 ## Target O/S
-
-The playbooks have been tested using Ansible 2.9.21 (or later) on target hosts with the following operating systems:
-
-* CentOS 7 and 8
-* Red Hat Enterprise Linux 7 and 8
-
-Additional target environments will be added in future releases.
+While Content Services supports a wide range of OS, the playbook is only supported for a subset of them. The table below 
+gives detailed information on the status of supported OS (Additional target environments will be added in future releases):
+
+| OS Flavor / version | 7.x Enterprise | 6.2.2 Enterprise | Community |
+|-|-|-|-|
+| RHEL 8.2 | [x] | - | [x] |
+| RHEL 7.7 | [x] | - | [x] |
+| RHEL 7.6 | [x] | [x] | [x] |
+| CentOS 8 x64 | [x] | - | [x] |
+| CentOS 7 x64 | [x] | [x] | [x] |
+| Ubuntu 20.04 | [x] | - | [x] |
+| Ubuntu 18.04 | [x] | [x] | [x] |
+
+>**Note:** Ansible version 2.12.x is used for testing this playbook.
```

Comment:
This will now be a strong requirement and is mentioned as such in PR #587 
Maybe it's worth removing this note from here